schemas/2015-07-01-preview/Microsoft.ServerManagement.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-07-01-preview/Microsoft.ServerManagement.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.ServerManagement",
+  "description": "Microsoft.ServerManagement Resource Types",
+  "resourceDefinitions": {
+    "gateway": {
+      "type": "object",
+      "description" : "A RSMT gateway",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2015-07-01-preview"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.ServerManagement/gateways"
+          ]
+        },
+        "location": {
```

Comment:
do not define location in RP schema, this could have conflict with the location defined in deploymentTemplate.json, just leave the verification of location to server.